modules/ROOT/pages/edifact-send-settings.adoc

```diff
@@ -13,10 +13,10 @@ To configure a partner's EDIFACT send settings:
 * <<character-set,Character set & delimiters>>
 . Click *Save*.
 
-The values you configure in the partner's settings page are used by default for all EDIFACT messages sent to that trading partner. You can override the settings you configure at the message flow level by selecting the custom option while creating an outbound message flow.
+The values you configure in the partner's settings page are used by default for all EDIFACT messages sent to that partner. You can override the settings you configure at the message flow level by selecting the custom option while creating an outbound message flow.
```

Comment:
The values in the partner's settings page are used by default for all EDIFACT messages sent to that partner. You can override the settings at the message flow level by selecting the custom option while creating an outbound message flow.